lib/msal-browser/src/event/EventHandler.ts

```diff
@@ -45,6 +49,28 @@ export class EventHandler {
         this.logger.verbose(`Event callback ${callbackId} removed.`);
     }
 
+    /**
+     * Adds event listener that emits an event when a user account is added or removed from localstorage in a different browser tab or window
+     */
+    addAccountStorageListener(): void {
+        if (!this.listeningToStorageEvents) {
+            this.logger.verbose("Adding account storage listener.");
+            this.listeningToStorageEvents = true;
+            window.addEventListener("storage", this.boundStorageListener);
+        }
```

Comment:
Can you add log statements in the `else` for both of these?

Reply:
Done